client/package.json

```diff
@@ -20,6 +20,8 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject"
   },
+  // This will allow the front end client to fetch data from the back end
+  "proxy":"process.env.PORT || http://localhost:5000",
```

Comment:
Sadly this won't work, you would have to use the localhost:5000 thing

Reply:
This should be resolved now.